src/settings/project/mod.rs

```diff
@@ -96,14 +62,23 @@ impl Project {
     }
 
     pub fn new() -> Result<Self, failure::Error> {
-        get_project_config()
+        let mut config_path = PathBuf::new();
+        config_path.push("./wrangler.toml");
+
+        get_project_config(config_path)
+    }
+
+    pub fn kv_namespaces(&self) -> Vec<KvNamespace> {
+        match &self.kv_namespaces {
+            Some(kv) => kv.clone(),
+            None => Vec::new(),
+        }
```

Comment:
this can be written

```rust
self.kv_namespaces.cloned().unwrap_or_else(Vec::new)
```

Reply:
i hope you meant `.clone()` bc `.cloned()` gave me many complaints.

Reply:
no, i meant cloned: https://doc.rust-lang.org/std/option/enum.Option.html#method.cloned

Reply:
![image](https://user-images.githubusercontent.com/27786/61984917-1d396280-afcc-11e9-9b9c-d30afae24230.png)

cloned goes from `Option<&T>` to `Option<T>`. But you already have an `Option<T>`. `clone` is right here, and I'm wrong.

Thanks, compiler!